docs/tutorial/tutorial.md

```diff
@@ -216,7 +216,7 @@ class Square extends React.Component {
 }
 ```
 
-In [JavaScript classes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes), you need to explicitly call `super();` when defining the constructor of a subclass.
+In [JavaScript classes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes), you need to explicitly call `super();` when defining the constructor of a subclass. When creating a class component in React, you should always call the base constructor with `props`.
```

Comment:
This isn't true though. Passing `props` to super is only required when access `this.props` in the constructor, which is pointless since you can just access `props` from the constructor args.